### PR TITLE
[linuxNew] add support for Ubuntu Impish

### PR DESCRIPTION
**Pull request by arthurbambou**

### CODE REVIEW

**Review by val-int1 — APPROVED** [review]

**Comment by karianna** [comment]
@gdams - I assume we're not going to support each and every 6 month release of Ubuntu?

File: linuxNew/jdk/debian/src/packageTest/java/packaging/DebianFlavours.java [hunk 1]
```diff
@@ -38,7 +38,7 @@
 			Arguments.of("debian", "bullseye"),
 			Arguments.of("ubuntu", "bionic"),
 			Arguments.of("ubuntu", "focal"),
-			Arguments.of("ubuntu", "hirsute")
```
Comment (gdams) [inline linuxNew/jdk/debian/src/packageTest/java/packaging/DebianFlavours.java:41]: You likely didn't intend to remove `hirsute` here?

Comment (arthurbambou) [reply linuxNew/jdk/debian/src/packageTest/java/packaging/DebianFlavours.java:41]: so you prefer to keep it?

Comment (gdams) [reply linuxNew/jdk/debian/src/packageTest/java/packaging/DebianFlavours.java:41]: Yes please

**Pull request closed without merging**